Update choosing-core-framework-server.md

vidyavrat opened the pull request:
Added WPF and WinForms in the list of most common technologies not found in .NET Core 1.0:

vidyavrat committed:
Update choosing-core-framework-server.md

Added WPF and WinForms in the list of most common technologies not found in .NET Core 1.0:

dnfclas commented:
This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

svick commented on docs/standard/choosing-core-framework-server.md:
```diff
@@ -89,6 +89,8 @@ Some .NET Framework technologies are not available in .NET Core. Some of them wi
 
 * Workflow related services: Windows Workflow Foundation (WF), Workflow Services (WCF + WF in a single service) and WCF Data Services (formerly known as “ADO.NET Data Services”) are only available on the .NET Framework and there are no plans to bring them to .NET Core.
 
+* WPF and Windows Forms: WPF (Windows Presentation Foundation) and Windows Forms Applications are only available on the .NET Framework and there are no plans to bring them to .NET Core and these will continue to be part of .NET Standard. 
```
> these will continue to be part of .NET Standard

WPF and Winforms are not part of .Net Standard.

vidyavrat committed:
Update choosing-core-framework-server.md

Fixed as per @svick comment.

GuardRex commented on docs/standard/choosing-core-framework-server.md:
```diff
@@ -89,6 +89,8 @@ Some .NET Framework technologies are not available in .NET Core. Some of them wi
 
 * Workflow related services: Windows Workflow Foundation (WF), Workflow Services (WCF + WF in a single service) and WCF Data Services (formerly known as “ADO.NET Data Services”) are only available on the .NET Framework and there are no plans to bring them to .NET Core.
 
+* WPF and Windows Forms: WPF (Windows Presentation Foundation) and Windows Forms Applications are only available on the .NET Framework and there are no plans to bring them to .NET Core and these will continue to be part of .NET Framework. 
```
It's unusual to state the abbreviation prior to the full name. "Applications" isn't a proper noun, so I suggest lowercase. Also, I recommend breaking the two thoughts into two sentences with a comma leading the conjunction between the independent clauses in the second sentence. I suggest something like "port" over "bring," which is more specific. Lastly, we're trying to go with present tense when possible/appropriate, so the "will continue" might become "remain."

> Windows Presentation Foundation (WPF) and Windows Forms: WPF and Windows Forms applications are only available on the .NET Framework. There are no plans to port them to .NET Core, and they remain part of the .NET Framework.

vidyavrat committed:
Update choosing-core-framework-server.md

Fixed as per @GuardRex PR Comment.

GuardRex commented on docs/standard/choosing-core-framework-server.md:
```diff
@@ -89,6 +89,8 @@ Some .NET Framework technologies are not available in .NET Core. Some of them wi
 
 * Workflow related services: Windows Workflow Foundation (WF), Workflow Services (WCF + WF in a single service) and WCF Data Services (formerly known as “ADO.NET Data Services”) are only available on the .NET Framework and there are no plans to bring them to .NET Core.
 
+* Windows Presentation Foundation (WPF) and Windows Forms: WPF and Windows Forms applications are only available on the .NET Framework. There are no plans to port them to .NET Core, and they remain part of the .NET Framework. 
```
I'd still recommend striking ", and they remain part of the .NET Framework", since it's stated in the first sentence. cc/ @mairaw

vidyavrat committed:
Update choosing-core-framework-server.md

Agreed, Fixed PR Comment from @GuardRex "striking ", and they remain part of the .NET Framework", since it's stated in the first sentence." @mairaw

GuardRex commented:
*Awesome!* Thank you @vidyavrat!

cartermp reviewed (approved):
LGTM.  Thanks, @vidyavrat!
